src/fsharp/NicePrint.fs

```diff
@@ -1301,6 +1301,29 @@ module private PrintTastMemberOrVals =
     let prettyLayoutOfMemberNoInstShort denv v = 
         prettyLayoutOfMemberShortOption denv emptyTyparInst v true |> snd
 
+    let prettyLayoutOfLiteralValue literalValue =
```

Comment:
@dsyme gave me similar feedback in a differen PR, since there is no type "prettying" done this should probably be called `layoutOfLiteralValue`

Reply:
changed to layoutOfLiteralValue